server/src/controllers/client/updateClient.js

```diff
@@ -0,0 +1,66 @@
+const logger = require("pino")();
+const Client = require("../../models/client.model");
+
+const updateClientRequestHandler = async (req, res) => {
+  try {
+    const client_id = req.params.client_id;
+    const client = await Client.findOne({ where: { id: client_id } });
+    if (!client) {
```

Comment:
When updating the client's status to active, we probably want to clear status_at_exit, status_at_x_months, as well as closure date